src/istio/values/config-passthrough.yaml

```diff
@@ -1,6 +1,7 @@
 name: passthrough
 
 tls:
+  supportTLSV1_2: false # set to true to enable TLSv1.2 support
```

Comment:
This addition made me realize our conditional around `minProtocolVersion` (checking that its not passthrough) probably shouldn't be there - I believe Istio allows the protocol version to be set on passthrough gateways as well.